source/brailleDisplayDrivers/albatross/driver.py

```diff
@@ -165,6 +169,8 @@ def _searchPorts(self, port: str):
 		"""
 		for self._baudRate in BAUD_RATE:
 			for portType, portId, port, portInfo in self._getTryPorts(port):
+				if not self._albatross_port(port):

```

Comment:
Something like this should be enough:

```suggestion
				if portinfo.get("busReportedDeviceDescription") != ALBATROSS_BUS_DEVICE_DESC :
```

Please also remove the stuff that's no longer necessary now.

Reply:
Nope I didn't. But then it would be:

```suggestion
				if portInfo.get("busReportedDeviceDescription", ALBATROSS_BUS_DEVICE_DES) != ALBATROSS_BUS_DEVICE_DESC:
```

or 

```suggestion
				if portInfo.get("busReportedDeviceDescription") in (ALBATROSS_BUS_DEVICE_DESC, None):
```